go.mod

```diff
@@ -12,11 +12,11 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/fatih/color v1.13.0
 	github.com/hashicorp/go-getter v1.5.11
-	github.com/hashicorp/go-hclog v1.1.0
+	github.com/hashicorp/go-hclog v1.2.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/hcl/v2 v2.10.1
-	github.com/hashicorp/nomad v1.2.7-0.20220223202307-7bcf0afd810b
-	github.com/hashicorp/nomad-openapi v0.0.0-20220408191729-79a1c29147b5
+	github.com/hashicorp/nomad v1.3.0-rc.1
+	github.com/hashicorp/nomad-openapi v0.0.0-20220623151231-ccfbbe2cdd4a
```

Comment:
Is this expected to be the release candidate? Any chance it can be v1.3.2 that was just released?

Reply:
This is not a release candidate. It is just a patch to unblock a user. But that said, it does make sense to update the upstream and then this again. I'll handle it Monday when I return.